local/src/core/http2.cc

```diff
@@ -833,6 +849,8 @@ on_data_chunk_recv_cb(
       stream_state._key,
       stream_id,
       TextView(reinterpret_cast<char const *>(data), len));
+  stream_state._body_received.reserve(stream_state._body_received.size() + len);
+  stream_state._body_received += std::string(reinterpret_cast<char const *>(data), len);
```

Comment:
Does this reserve help much here? We reserve the space, then append a string of that same space. I would not think that saves much.

I wonder whether we can reserve the replay content size at transaction construction. If we could do that, that would be helpful.

(This comment also applies to http3.)

Reply:
I'm using the `content-length` header to reserve the size now.